README.md

```diff
@@ -144,12 +141,26 @@ When keys are generated, you can configure your DNS server by just pasting the c
 
 #### Custom user changes & patches
 
-If you'd like to change, patch or alter files or behavior of `docker-mailserver`, you can use a script. See the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/user-patches/) for a detailed explanation.
+If you'd like to change, patch or alter files or behavior of `docker-mailserver`, you can use a script. See the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/override-defaults/user-patches/) for a detailed explanation.
+
+#### Updating `docker-mailserver`
+
+``` BASH
+docker-compose down
+docker pull docker.io/mailserver/docker-mailserver:<VERSION TAG>
```

Comment:
It doesn't matter which version you pull. docker-compose will only use the configured version set in the yml configuration when starting a container.
Therefore I would change this to a simple: `docker-compose pull.`

To reduce the downtime, I would also change the order to:

```
docker-compose pull
docker-compose down
docker-compose up -d mailserver
```

Reply:
Good point. Feel free to change this - I'm all for it.